pages/recipes/steak_tartare.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="/style/style.css" rel="stylesheet">
```

Comment:
Careful with your file paths! The `/style/style.css` path does not actually work because it is an incorrect absolute path. Starting a path with a slash (/) will start the file search at the root directory of your entire file system (not your project).
To correct this, we can use a relative path instead. This will provide directions to the target file based on the current file's location.
In code this looks like: `../../style/style.css`. The two dots at the beginning signal to start searching in the parent directory of the current directory. Since this directory is embedded two levels down from the home directory, we need two of them.
Here is a great resource on absolute vs relative paths:  https://www.geeksforgeeks.org/absolute-relative-pathnames-unix/

This applies to image URLs as well. On these recipe pages, the CSS styles and the pictures do not show up.